webpack/assets/javascripts/react_app/components/Layout/components/TaxonomySwitcher/TaxonomyDropdown.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { ContextSelectorItem } from '@patternfly/react-core';
+import CustomContextSelector from '../CustomContextSelector';
+import { noop } from '../../../../common/helpers';
+import { translate as __ } from '../../../../common/I18n';
+import './TaxonomyDropdown.scss';
+
+const TaxonomyDropdown = ({
+  taxonomyType,
+  currentTaxonomy,
+  taxonomies,
+  changeTaxonomy,
+  anyTaxonomyText,
+  manageTaxonomyText,
+  anyTaxonomyURL,
+  manageTaxonomyURL,
+  ...props
+}) => {
+  const [searchValue, setSearchValue] = useState('');
+  const [isOpen, setIsOpen] = useState(false);
+  const [filteredItems, setFilteredItems] = useState(taxonomies);
+
+  const onToggle = (event, newIsOpen) => {
+    setIsOpen(newIsOpen);
+  };
+  const onSelect = () => {
+    setIsOpen(!isOpen);
+  };
+  const onSearchInputChange = (value, event) => {
+    setSearchValue(event.target.value);
+    onSearchButtonClick();
+  };
+  const onSearchButtonClick = () => {
+    const filtered =
+      searchValue === ''
+        ? taxonomies
+        : taxonomies.filter(item =>
+            item.title.toLowerCase().includes(searchValue.toLowerCase())
+          );
+    setFilteredItems(filtered || []);
+  };
+
+  const staticGroup = {
+    title: __(taxonomyType),
+    items: [
+      {
+        title: __(anyTaxonomyText),
+        href: anyTaxonomyURL,
+        onClick: () => {
+          changeTaxonomy({ title: anyTaxonomyText });
+        },
+        className: `${taxonomyType.toLowerCase()}s_clear`,
+      },
+      {
+        title: __(manageTaxonomyText),
+        href: manageTaxonomyURL,
+        className: taxonomyType.toLowerCase(),
+      },
+    ],
+  };
+  return (
+    <CustomContextSelector
+      toggleText={currentTaxonomy}
+      onSearchInputChange={onSearchInputChange}
+      isOpen={isOpen}
+      searchInputValue={searchValue}
+      onToggle={onToggle}
+      onSelect={onSelect}
+      onSearchButtonClick={onSearchButtonClick}
+      screenReaderLabel="Selected Taxonomy:"
+      showFilter={taxonomies.length > 6}
```

Comment:
can we always display it?
or from `taxonomies.length > 1` ?

Reply:
https://github.com/theforeman/foreman/blob/cbf3a1f127849a8056aa13d772186e290ab19b13/webpack/assets/javascripts/react_app/components/Layout/components/TaxonomyDropdown.js#L67
[link to code](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/Layout/components/TaxonomyDropdown.js#L67)

Why should we change existing behavior?

Reply:
ok then, let's stick with the 6 items :+1: